examples/guided.py

```diff
@@ -308,31 +407,45 @@ def perform_installation(mountpoint):
 	Only requirement is that the block devices are
 	formatted and setup prior to entering this function.
 	"""
-	with archinstall.Installer(mountpoint, kernels=archinstall.arguments.get('kernels', 'linux')) as installation:
+	with archinstall.Installer(
+		mountpoint, kernels=archinstall.arguments.get('kernels', 'linux')
+	) as installation:
 		## if len(mirrors):
 		# Certain services might be running that affects the system during installation.
 		# Currently, only one such service is "reflector.service" which updates /etc/pacman.d/mirrorlist
 		# We need to wait for it before we continue since we opted in to use a custom mirror/region.
-		installation.log(f'Waiting for automatic mirror selection (reflector) to complete.', level=logging.INFO)
+		installation.log(
+			f'Waiting for automatic mirror selection (reflector) to complete.',
+			level=logging.INFO,
+		)
 		while archinstall.service_state('reflector') not in ('dead', 'failed'):
 			time.sleep(1)
 		# Set mirrors used by pacstrap (outside of installation)
 		if archinstall.arguments.get('mirror-region', None):
-			archinstall.use_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors for the live medium
+			archinstall.use_mirrors(
+				archinstall.arguments['mirror-region']
+			)  # Set the mirrors for the live medium
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
-			if archinstall.arguments['mirror-region'].get("mirrors",{})!= None:
-				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
-			if archinstall.arguments["bootloader"]=="grub-install" and hasUEFI()==True:
+			if archinstall.arguments['mirror-region'].get("mirrors", {}) != None:
+				installation.set_mirrors(
+					archinstall.arguments['mirror-region']
+				)  # Set the mirrors in the installation medium
+			if archinstall.arguments["bootloader"] == "grub-install" and hasUEFI() == True:
 				installation.add_additional_packages("grub")
 			installation.set_keyboard_language(archinstall.arguments['keyboard-language'])
 			installation.add_bootloader(archinstall.arguments["bootloader"])
 
 			# If user selected to copy the current ISO network configuration
 			# Perform a copy of the config
-			if archinstall.arguments.get('nic', {}) == 'Copy ISO network configuration to installation':
-				installation.copy_ISO_network_config(enable_services=True) # Sources the ISO network configuration to the install medium.
-			elif archinstall.arguments.get('nic', {}).get('NetworkManager',False):
+			if (
+				archinstall.arguments.get('nic', {})
+				== 'Copy ISO network configuration to installation'
+			):
+				installation.copy_ISO_network_config(
+					enable_services=True
+				)  # Sources the ISO network configuration to the install medium.
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.